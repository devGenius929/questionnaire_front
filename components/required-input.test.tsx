import React from 'react';
import renderer from 'react-test-renderer';
import RequiredInput from './required-input';

describe('Menu', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <RequiredInput
          id="Name"
          name="Name"
          className="Name"
          label="Nome*"
          value="Jo Blake"
          onChange={(e) => console.log('Changed!')}
        />,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
