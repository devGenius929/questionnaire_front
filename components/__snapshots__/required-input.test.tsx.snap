// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu renders correctly 1`] = `
<div
  className="formcel1col"
>
  <label
    className="Name"
  >
    Nome*
  </label>
  <input
    className="Name"
    id="Name"
    name="Name"
    onChange={[Function]}
    type="text"
    value="Jo Blake"
  />
</div>
`;
